action.yml

```diff
@@ -6,6 +6,16 @@ branding:
   color: red
 
 inputs:
+  github_pat:
+    description: |
+      Github personal access token.
+      This is required if tools need to be installed from GitHub.
+      This will be used to access GitHub API's, as authenticated API calls have more quota to hit api rate limit.
+    required: false
+  source:
+    description: Source from which to download the tool, accepted values are 'github' or 'mirror'
```

Comment:
Source from which to download all tools in this step. Can be "github" or "mirror".